lib/l10n/app_en.arb

```diff
@@ -1308,5 +1308,23 @@
     "description": "Text shown on button to require an item which is already downloaded by a parent collection, preventing it from being deleted if the parent is removed."
   },
   "showArtistChipImage": "Show artist images with artist name",
-  "showArtistChipImageSubtitle": "This affects small artist image previews, such as on the player screen."
+  "showArtistChipImageSubtitle": "This affects small artist image previews, such as on the player screen.",
+  "queueTotalSize": "{ trackPosition } / { trackCount }  ({hoursRemaining, select, 0{} other{{hoursRemaining}h }}{minutesRemaining}m remaining)",
```

Comment:
I'd prefer not having the index an total size (`{trackPosition } / { trackCount}`) as a translatable string. I don't really see a reason why this would need to be translated (not sure about RTL languages though), and I can already see the translations messing up the formatting by omitting one of the spaces or something similar. Everything inside the parentheses should be sufficient, with a more fitting name (like `remainingDuration`).

Or was there a deeper reason for putting it all together like this?

Reply:
I didn't have a specific reason, I'm just not sure how much flexibility translations might need and so just put everything in.  The spacing to make the trackcount looks right doesn't seem to be more complicated than the spacing needed to make some of the fancier select cases look right.  I think a RTL language would reverse the whole thing, which might be the desired behavior?  It's questions like this that made me put as much of the string into the translation as possible, but I can simplify it if you think that's best.

Reply:
Right, I'm not sure either. The selects aren't as prominent, that's why I'm less concerned there.  
What you could do is move them to a separate translation string at least, so that they're independent and can be reused more easily?

Reply:
I'm not entirely sure how useful that would be?  I've decided to just implement your original suggestion.

Reply:
We'll that would still support reversing the order for RTL languages.

But it's fine for now, there's no way to translate beta strings yet anyway. We can deal with it if it actually causes issues :)